Components/ShareMovieButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Share, View } from 'react-native';
+import { ShareButton, ShareText } from '../styled-components/styles';
+
+const ShareMovieButton = ({ movieToShare }) => {
+  const onShare = async () => {
```

Comment:
I'm impressed, great job with using `async await` :)
For deeper explanation, you can check out https://medium.com/javascript-in-plain-english/async-await-javascript-5038668ec6eb